app/models/manageiq/providers/ansible_tower/shared/automation_manager/configuration_script_source.rb

```diff
@@ -14,7 +14,7 @@ def create_in_provider(manager_id, params)
 
     def create_in_provider_queue(manager_id, params)
       manager = ExtManagementSystem.find(manager_id)
-      action = "Creating #{name} with name=#{params[:name]}"
+      action = "Creating #{name} with name=#{params[:name] || params['name']}"
```

Comment:
Hrm, I saw this in another PR.  I don't like the fact that we don't know whether the param keys will be strings or symbols.  That seems like an easy thing to fix.  The MIQ UI and API should be sending param keys consistently.